Makefile

```diff
@@ -3,8 +3,9 @@ MAKEFLAGS+=-r
 
 config=debug
 defines=standard
+cxxstd=default
```

Comment:
> So generally I'm the only person using the Makefile :)

Ok, I thought of people who download pugixml, compile it and run the tests to see that it works. If compiling fails (and may it be only because the compiler does not know `-std=c++11`), it may distract some people of using pugixml for their projects.

That was the last point I make in favor of a "do not use `-std`"-default.
If you really want the default to be c++11, I change my branch. However, I'd say that there should still be a magic setting `cxxstd=any` or something to disable `-std` usage.